code/string-algorithms/anagram_search/anagram_search.java

```diff
@@ -0,0 +1,34 @@
+/* Part of Cosmos by OpenGenus Foundation */
+import java.util.Arrays;  
+import java.util.Scanner;  
+       
+    public class anagram_search {  
```

Comment:
Code is pretty cool 👍      
But the  class name does not follow the camel casing rule ......kindly look into it .......Well by changing the class name you will have to change the file name also !
Please write class name as  :-     Anagram_Search 
Please write file name as :-      Anagram_Search.java ! 👍

Reply:
It doesn't have to be camel case, it just has to be consistent (Which it is not)

Reply:
I don't know exactly what is meant by consistent here..........but still https://www.javatpoint.com/java-naming-conventions this is a good reference ! 👍 ...may kindly check ! :)

Reply:
@yatharthshahjpr By consistency I meant that in some cases Camel Case is used, while in other underscore case. The author needs to pick one.

Reply:
Any conclusion guys, changes required?? @yatharthshahjpr  and @arnavb